rollout/replicaset.go

```diff
@@ -120,10 +119,53 @@ func (c *rolloutContext) reconcileNewReplicaSet() (bool, error) {
 	if err != nil {
 		return false, err
 	}
+
+	abortScaleDownDelaySeconds := time.Duration(defaults.GetAbortScaleDownDelaySecondsOrDefault(c.rollout))
+	if c.isScaleDownOnabort() {
+		c.log.Info("scale down on abort for the newRS")
+
+		// if the newRS has scale down annotation and to be scaled down now, set
```

Comment:
Is the comma supposed to be after set and not now at the end of the comment?

Reply:
Updated the log message. Thanks.